handlers/posterior_matching.py

```diff
@@ -32,7 +43,7 @@ def initializePosterior(self):
         self.pR = self.pR / np.sum(self.pR)
         self.calculateNextIntensity()
 
-    def addResponse(self, result, intensity=None):
+    def addResponse(self, task, result, intensity=None):
```

Comment:
No.

This breaks the API, do not do this.

We can set the task name for the handler at initialization, we don't have to pass it in every time.

Also, instead of hardcoding the alpha values in here, we can pass them in at initialization instead.